config/packages/security.yaml

```diff
@@ -41,6 +42,7 @@ security:
         new_api_admin_user:
             pattern: "%sylius.security.new_api_admin_regex%/.*"
             provider: sylius_api_admin_user_provider
+            user_checker: security.user_checker.chain.new_api_admin_user
```

Comment:
This is not related to this PR, but we should change firewall names in the API context,
preferably dropping the `new` and `user` parts

Reply:
I thought the same thing at the first review of this PR 👍🏻

Reply:
#17240